### PR TITLE
[docs] window size for locreg detrending

### PR DESCRIPTION
**Pull request by vasileermicioi**
New PR as a follow-up of the discussion here https://github.com/neuropsychology/NeuroKit/pull/734

### CODE REVIEW

File: neurokit2/signal/signal_detrend.py [hunk 1]
```diff
@@ -43,8 +43,10 @@ def signal_detrend(
         Only used if ``method`` is "loess". The parameter which controls the degree of smoothing.
     window : float
         Only used if ``method`` is "locreg". The detrending ``window`` should correspond to the
-        desired low frequency band to remove (for instance, ``1.5`` will remove frequencies below
-        1.5Hz).
+        sample rate divided by the desired low-frequency band to remove
```
Comment (DominiqueMakowski) [inline neurokit2/signal/signal_detrend.py:46]:
```suggestion
        sampling rate divided by the desired low-frequency band to remove 
```

File: neurokit2/signal/signal_detrend.py [hunk 2]
```diff
@@ -43,8 +43,10 @@ def signal_detrend(
         Only used if ``method`` is "loess". The parameter which controls the degree of smoothing.
     window : float
         Only used if ``method`` is "locreg". The detrending ``window`` should correspond to the
-        desired low frequency band to remove (for instance, ``1.5`` will remove frequencies below
-        1.5Hz).
+        sampling rate divided by the desired low-frequency band to remove 
+        (``window = sample_rate / detrend_frequency``).
```
Comment (danibene) [inline neurokit2/signal/signal_detrend.py:47]:
```suggestion
        (``window = 1 / detrend_frequency``)
```

File: neurokit2/signal/signal_detrend.py [hunk 3]
```diff
@@ -43,8 +43,10 @@ def signal_detrend(
         Only used if ``method`` is "loess". The parameter which controls the degree of smoothing.
     window : float
         Only used if ``method`` is "locreg". The detrending ``window`` should correspond to the
-        desired low frequency band to remove (for instance, ``1.5`` will remove frequencies below
-        1.5Hz).
+        sampling rate divided by the desired low-frequency band to remove
```
Comment (danibene) [inline neurokit2/signal/signal_detrend.py:46]:
```suggestion
        1 divided by the desired low-frequency band to remove
```

File: neurokit2/signal/signal_detrend.py [hunk 4]
```diff
@@ -43,8 +43,10 @@ def signal_detrend(
         Only used if ``method`` is "loess". The parameter which controls the degree of smoothing.
     window : float
         Only used if ``method`` is "locreg". The detrending ``window`` should correspond to the
-        desired low frequency band to remove (for instance, ``1.5`` will remove frequencies below
-        1.5Hz).
+        sampling rate divided by the desired low-frequency band to remove 
+        (``window = sample_rate / detrend_frequency``).
+        For instance, to remove frequencies below ``0.67Hz`` the window should be ``1500``
```
Comment (danibene) [inline neurokit2/signal/signal_detrend.py:48]:
```suggestion
        For instance, to remove frequencies below ``0.67Hz`` the window should be ``1.5``
```

File: neurokit2/signal/signal_detrend.py [hunk 5]
```diff
@@ -43,8 +43,10 @@ def signal_detrend(
...
+        (``1000 / 0.67 = 1500``).
```
Comment (danibene) [inline neurokit2/signal/signal_detrend.py:49]:
```suggestion
        (``1 / 0.67 = 1.5``).
```

File: studies/ecg_benchmark/README.md [hunk 6]
```diff
@@ -524,7 +524,7 @@ def tarvainen(ecg, sampling_rate):
 def locreg(ecg, sampling_rate):
     ecg = nk.signal_detrend(ecg, 
                             method="locreg", 
-                            window=0.5*sampling_rate, 
+                            window=sampling_rate/0.5,
```
Comment (danibene) [inline studies/ecg_benchmark/README.md:527]:
```suggestion
                            window=1/0.5,
```

File: studies/ecg_benchmark/make_data.py [hunk 7]
```diff
@@ -152,7 +152,7 @@ def standardization(ecg, sampling_rate):
 #def locreg(ecg, sampling_rate):
 #    ecg = nk.signal_detrend(ecg,
 #                            method="locreg",
-#                            window=0.5*sampling_rate,
+#                            window=sampling_rate/0.5,
```
Comment (danibene) [inline studies/ecg_benchmark/make_data.py:155]:
```suggestion
#                            window=1/0.5,
```

File: studies/ecg_benchmark/make_data.py [hunk 8]
```diff
@@ -152,7 +152,7 @@ def standardization(ecg, sampling_rate):
...
 #                            stepsize=0.02*sampling_rate)
```
Comment (danibene) [inline studies/ecg_benchmark/make_data.py:156]:
```suggestion
#                            stepsize=0.02)
```

File: studies/ecg_benchmark/README.md [hunk 9]
```diff
@@ -524,7 +524,7 @@ def tarvainen(ecg, sampling_rate):
 def locreg(ecg, sampling_rate):
     ecg = nk.signal_detrend(ecg, 
                             method="locreg", 
-                            window=0.5*sampling_rate, 
+                            window=sampling_rate/0.5, 
                             stepsize=0.02*sampling_rate)
```
Comment (danibene) [inline studies/ecg_benchmark/README.md:528]:
```suggestion
                            stepsize=0.02)
```

**Review by danibene — CHANGES_REQUESTED** [review]
I think we shouldn't multiply `window` or `stepsize` by the sample rate because that's already done within the function

**Comment by vasileermicioi** [comment]
if I don't multiply by sampling_rate I get an error `'stepsize' should be more than 1. Increase its value`

`nk.signal_detrend(sig, method="locreg", window=0.5, stepsize=0.1)`

`nk.__version__` is 0.2.1

**Comment by danibene** [comment]
> if I don't multiply by sampling_rate I get an error `'stepsize' should be more than 1. Increase its value`
> 
> `nk.signal_detrend(sig, method="locreg", window=0.5, stepsize=0.1)`
> 
> `nk.__version__` is 0.2.1

@vasileermicioi have you tried with the dev branch? That the sampling rate is multiplied within the function is a new change since 0.2.1

**Comment by DominiqueMakowski** [comment]
@danibene perhaps if it's easier (instead of having @vasileermicioi fiddling with the branches) you can commit the edits and merge?

**Pull request merged**